tests/tests_app_examples/test_commands_and_api.py

```diff
@@ -1,31 +1,37 @@
 import os
 from subprocess import Popen
 from time import sleep
-from unittest import mock
 
 import pytest
+import requests
 from tests_app import _PROJECT_ROOT
 
 from lightning_app.testing.testing import run_app_in_cloud
 
 
-@mock.patch.dict(os.environ, {"SKIP_LIGHTING_UTILITY_WHEELS_BUILD": "0"})
 @pytest.mark.cloud
-def test_commands_example_cloud() -> None:
-    with run_app_in_cloud(os.path.join(_PROJECT_ROOT, "examples/app_commands")) as (
+def test_commands_and_api_example_cloud() -> None:
+    with run_app_in_cloud(os.path.join(_PROJECT_ROOT, "examples/app_commands_and_api")) as (
         admin_page,
-        _,
+        view_page,
         fetch_logs,
     ):
         app_id = admin_page.url.split("/")[-1]
-        cmd = f"lightning trigger_with_client_command --name=something --app_id {app_id}"
+        cmd = f"lightning command_with_client --name=this --app_id {app_id}"
         Popen(cmd, shell=True).wait()
-        cmd = f"lightning trigger_without_client_command --name=else --app_id {app_id}"
+        cmd = f"lightning command_without_client --name=is --app_id {app_id}"
         Popen(cmd, shell=True).wait()
 
+        sleep(10)
```

Comment:
Is there a way to leverage the readiness of the app (+ eventual timeout) in order to avoid this test to be flaky and take more time than needed in order to be conservative?